packages/sirius-web/frontend/sirius-web-application/src/views/new-project/NewProjectView.tsx

```diff
@@ -103,6 +105,7 @@ export const NewProjectView = () => {
   const { name, nameMessage, nameIsInvalid, message, newProjectId } = context;
   const [createProject, { loading, data, error }] = useMutation<GQLCreateProjectMutationData>(createProjectMutation);
   const { Component: Footer } = useComponent(footerExtensionPoint);
+  const { canCreate } = useContext<ProjectsCapabilitiesContextValue>(ProjectsCapabilitiesContext);
```

Comment:
This should leveraged a hook named `useCurrentViewer()` to retrieve the viewer. We should thus have a `ViewerContext` and a `useViewer()` to retrieve it. We should probably do that before the `Router` to be able to deny some routes entirely.

Reply:
Done